packages/block-editor/src/components/line-height-control/index.js

```diff
@@ -16,6 +16,8 @@ import {
 } from './utils';
 
 const LineHeightControl = ( {
+	/** Start opting into the larger default height that will become the default size in a future version. */
+	__next40pxDefaultSize = false,
```

Comment:
This was already being passed through via `otherProps`, but making it explicit here. All in-app usage already uses the correct size.

Reply:
Thought: instead of exposing `__next40pxDefaultSize`, could we instead just set `__next40pxDefaultSize` to `true` on `NumberControl` ? Or do we still care about offering consumers the option to opt-out of the new size, for the time being?

Reply:
I considered it, but checked third-party usage of some of these Typography controls and decided against it. They were used more than I thought, and some of them still use them in the old sizes, so decided against it.